Security Checklist (#194)
crypto-org-chain/chain-docs: docs/getting-started/security-checklist.md

# Crypto.com Mainnet Validator Security Checklist

## Part 1 - Conduct Survey on General Controls of Hosting Data Centre

> Description: Perform a survey on the hosting data centre, and compare your result with the best practice below

Author: ```suggestion
Description: Perform a survey on the hosting data centre, and compare your result with the best practice suggested below
```

For example, your hosting data centre should have following features

| Controls Category | Description of Best Practice    |
| ----------------- | ------------------------------- |
| Data Center       | Redundant Power                 |
| Data Center       | Redundant Cooling               |
| Data Center       | Redundant Networking            |
| Data Center       | Physical Cage/Gated Access      |
| Data Center       | Remote Alerting Security Camera |

## Part 2 - Current Status of Node Setup

> Description: Perform a survey on your current status of node setup , and compare your result with the best practice belo

Author: ```suggestion
Description: Perform a survey on your current status of node setup, and compare your result with the best practice suggested below
```

| Redundancy                              | System monitoring and alerting is setup to alert owners on anormalies                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
| Key Managment                           | Setup Tendermint KMS with HSM or equivalent online service, which should replace the static key file.                                                                                                                                                                                                                                                                                                                                                                                                                                   |

> | DDOS                                    | Setup validator in accordance with sentry architecture. Setup [instruction](https://docs.tendermint.com/master/nodes/validators.html#setting-up-a-validator) and [detailed description](https://forum.cosmos.network/t/sentry-node-architecture-overview/454).                                                                                                                                                                                                                                                                          |

Author: ```suggestion
| DDOS                                    | Setup validator in accordance with sentry architecture. Kindly refer to the setup [instruction](https://docs.tendermint.com/master/nodes/validators.html#setting-up-a-validator) and [detailed description](https://forum.cosmos.network/t/sentry-node-architecture-overview/454).                                                                                                                                                                                                                                                                          |
```
